CMakeLists.txt

```diff
@@ -417,6 +432,16 @@ if(USE_MAYA)
   include(OpenVDBMayaSetup)
 endif()
 
+# Configure malloc library. Use Tbbmalloc for Windows or Maya and Jemalloc otherwise.
```

Comment:
Update this comment to include the macos decision with a reference to the problem (as looks like we can actually use jemalloc on mac after a specific version). Also perhaps expand on why windows uses tbbmalloc too for future reference.

Reply:
Done.